packages/presentation/lib/screen/splash/splash_bloc.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:async';
+import 'package:presentation/base/bloc.dart';
+import 'package:presentation/screen/home/home_screen.dart';
+import 'package:presentation/screen/splash/splash_data.dart';
+
+abstract class SplashBloc extends Bloc {
+  factory SplashBloc() => SplashBlocImpl();
+}
+
+class SplashBlocImpl extends BlocImpl implements SplashBloc {
+  var _state = SplashData.init();
+
+  @override
+  void initState() {
+    super.initState();
+    _updateData(data: _state);
+    Future.delayed(
```

Comment:
to imitate api cal, u should to use UseCase (C) Sergey

Reply:
corrected